test/python/transpiler/test_unitary_synthesis.py

```diff
@@ -570,6 +574,34 @@ def test_coupling_map_transpile(self, opt):
             )
         )
 
+    @combine(
+        opt_level=[0, 1, 2, 3],
+        bidirectional=[True, False],
+        dsc=(
+            "test natural_direction works with transpile using opt_level {opt_level} on"
+            "target with multiple 2q gates with bidirectional={bidirectional}"
+        ),
+        name="opt_level_{opt_level}_bidirectional_{bidirectional}",
+    )
+    def test_coupling_map_transpile_with_backendv2(self, opt_level, bidirectional):
+        backend = FakeBackend5QV2(bidirectional)
+        qr = QuantumRegister(2)
+        circ = QuantumCircuit(qr)
+        circ.append(random_unitary(4, seed=1), [0, 1])
+        circ_01 = transpile(
+            circ, backend=backend, optimization_level=opt_level, layout_method="trivial"
+        )
+        circ_01_index = {qubit: index for index, qubit in enumerate(circ_01.qubits)}
+        self.assertGreaterEqual(len(circ_01.get_instructions("cx")), 1)
+        self.assertTrue(
+            all(
+                (
+                    (0, 1) == (circ_01_index[qlist[0]], circ_01_index[qlist[1]])
+                    for _, qlist, _ in circ_01.get_instructions("cx")
+                )
+            )
+        )
```

Comment:
Super minor, but it wouldn't hurt to reformat this as
```python
for _, qlist, _ in circ_01.get_instructions("cx"):
    self.assertEqual((0, 1), ...)
```
so that if it fails, the error message will show what the first failing value was. (etc in other tests, though if this was pre-existing in this file, no need to do any of this in this PR)

Reply:
Sure thing, I was just copying the tests in here already but this is a better way to do it

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7795/commits/b570f299740e181f3748a8631509278257391365